Script/install.sh

```diff
@@ -1,5 +1,4 @@
 #! /bin/bash
-curl --silent https://raw.githubusercontent.com/DexterInd/script_tools/master/install_script_tools.sh | bash
```

Comment:
You've removed the curl install_script_tools.sh 
Is it done elsewhere? Cause you can't assume it will be on the SD card if people run this script directly.

Reply:
Since `install.sh` is such a mess, it shouldn't be used for installing the GrovePi. Therefore, installing the GrovePi only works by using the command for updating the repo. All after, there's a need for internet access in both cases. 

I have moved this out of here, because it now resides in `update_grovepi.sh`. It's not okay to call it twice in an update - `update_grovepi.sh` is used when we do the curl stuff.